src/diffusers/pipelines/kandinsky/pipeline_kandinsky_prior.py

```diff
@@ -116,14 +116,14 @@ class KandinskyPriorPipelineOutput(BaseOutput):
     Output class for KandinskyPriorPipeline.
 
     Args:
-        images (`torch.FloatTensor`)
+        image_embeds (`torch.FloatTensor`)
             clip image embeddings for text prompt
-        zero_embeds (`List[PIL.Image.Image]` or `np.ndarray`)
+        negative_image_embeds (`List[PIL.Image.Image]` or `np.ndarray`)
             clip image embeddings for unconditional tokens
     """
 
-    images: Union[torch.FloatTensor, np.ndarray]
```

Comment:
Think embeddings != images. Would be in favor of slightly renaming the output here so that one knows exactly what prior output is what t2i input